libsodium version preventing 8.1.1 compiling

**Describe the bug**
Using the latest scripts it was not possible to get the node to compile without separately building a new version of libsodium.
The version provided by the scripts does not work.

To correct the issue

```
cd git
git clone https://github.com/input-output-hk/libsodium
cd libsodium
git fetch --tags --all
git checkout dbb48cc
./autogen.sh
./configure
make
sudo make install
```


Then follow build instructions


**To Reproduce**
Steps to reproduce the behavior:
1. Follow guild instructions and attempt to compile node
2. Fails when building cardano node
3. Install correct libsodium version
4. Build successfully
5. See error from step 2
```
home/cardano/.cabal/store/ghc-8.10.7/cardano-crypto-praos-2.1.2.0-3665a2444452565d86335fe527ffb30a40b8f0c82ad16f51f7ad140e0105c385/lib/libHScardano-crypto-praos-2.1.2.0-3665a2444452565d86335fe527ffb30a40b8f0c82ad16f51f7ad140e0105c385.a(PraosBatchCompat.o):ghc_18.c:function cardanozmcryptozmpraoszm2zi1zi2zi0zm3665a2444452565d86335fe527ffb30a40b8f0c82ad16f51f7ad140e0105c385_CardanoziCryptoziVRFziPraosBatchCompat_zdwskFromBytes_info: error: undefined reference to 'crypto_vrf_ietfdraft13_secretkeybytes'
collect2: error: ld returned 1 exit status
`gcc' failed in phase `Linker'. (Exit code: 1)
cabal: Failed to build exe:cardano-cli from cardano-cli-8.1.1.
Failed to build exe:cardano-node from cardano-node-8.1.1.
Failed to build exe:cardano-submit-api from cardano-submit-api-3.1.2.
```

**Expected behavior**
Successful build

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Version:**
 - OS: Ubuntu 20.04LTS
 - Product version: n/a (latest from repo)
 - Cardano Node version: 8.1.1
 - Network you're connecting to: n/a

**Additional context**
Attempted to download correct libsodium in advance of build using:
./guild-deploy.sh -n preprod -s pbl

--- a/scripts/cnode-helper-scripts/guild-deploy.sh
+++ b/scripts/cnode-helper-scripts/guild-deploy.sh
@@ -281,6 +281,7 @@ build_dependencies() {
   pushd "${HOME}"/git >/dev/null || err_exit
   [[ ! -d "./secp256k1" ]] && git clone https://github.com/bitcoin-core/secp256k1 &>/dev/null
   pushd secp256k1 >/dev/null || err_exit
+  git fetch >/dev/null 2>&1
   git checkout ac83be33 &>/dev/null
   ./autogen.sh > autogen.log > /tmp/secp256k1.log 2>&1
   ./configure --enable-module-schnorrsig --enable-experimental > configure.log >> /tmp/secp256k1.log 2>&1
@@ -304,6 +305,7 @@ build_libsodium() {
   pushd "${HOME}"/git >/dev/null || err_exit
   [[ ! -d "./libsodium" ]] && git clone https://github.com/input-output-hk/libsodium &>/dev/null
   pushd libsodium >/dev/null || err_exit
+  git fetch >/dev/null 2>&1
   git checkout dbb48cc &>/dev/null
   ./autogen.sh > autogen.log > /tmp/libsodium.log 2>&1
   ./configure > configure.log >> /tmp/libsodium.log 2>&1

